tests/test_twisted.py

```diff
@@ -184,6 +184,19 @@ def test_catchesKeyCertificateMismatch(self, tmpdir):
 
         assert str(excinfo.value).startswith("No certificate matching ")
 
+    def test_catchesMultipleDHParams(self, tmpdir):
```

Comment:
Blah sorry, tried to page too many projects into memory at once and some of the old ones got paged out again ;)